### DISCUSSION TITLE
Welcome to goc Discussions!

### DISCUSSION THREAD

**Discussion by CarlJi** [qiniu/goc#138] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by debbyzang** [comment]
step 2 :goc build .  (or goc install )
with error ：
[root@localhost projectdemo]# goc install

FATA Fail to install: fail to generate new go.mod: open /tmp/goc-build-52be089fc042/go.mod: no such file or directory

ps：which path shoud I run ：go run .  (saw it on the flash) ?
